plugins/labhub.py

```diff
@@ -302,6 +302,14 @@ def difficulty_level(user, iss):
                 elif self.GH3_ORG.is_member(user):
                     return True
 
+            @register_check
+            def block_gci_issue_assignment(user, iss):
+                """
+                True if the issue is not labelled with 'initiatives/gci'.
+                False if the issue has been labelled with 'initiatives/gci'.
+                """
+                return 'initiatives/gci' not in iss.labels
```

Comment:
This doesnt implement allowing GCI participants to assign these issues.

Reply:
But, it blocks gci issue assignment using corobo to all users.

Reply:
@jayvdb I guess it is good for now, the issue https://github.com/coala/corobo/issues/501 doesn't specify that gci user can assign himself via corobo